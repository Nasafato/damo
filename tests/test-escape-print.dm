def main() : int
{
    print("Hello, literal world! \n\n Jello World");
    return 0;
}
