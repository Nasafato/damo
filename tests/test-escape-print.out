Hello, literal world!

 Jello World
